.gitignore

```diff
@@ -16,3 +16,5 @@ code/reports/mutation
 # Mock backend
 mock-backend/node_modules
 
+# Environment variables
+code/.env.local
```

Comment:
better add all `.env` files to `.gitignore`

Reply:
Thanks for suggestion. I used `.env*` to add all `.env` files to gitignore.